src/lib/routing-table.js

```diff
@@ -35,6 +35,28 @@ class RoutingTable {
     return false
   }
 
+  /**
+   * Compute a curve quote's `appliesToPrefix`, which is the shortest prefix
+   * that uniquely matches the target.
+   *
+   * @param {IlpAddress} routePrefix
+   * @param {IlpAddress} destinationAccount
+   * @returns {IlpAddress}
+   */
+  getAppliesToPrefix (routePrefix, destinationAccount) {
```

Comment:
Example:

A connector has 2 routes: one with `targetPrefix="abc."` and one with `targetPrefix=""`.
* If you request a quote to `"abc.def.alice"`, the best `appliesToPrefix` you can get is `"abc."`.
* If you request a quote to `"random.alice"`, it matches `targetPrefix=""`, but the best `appliesToPrefix` possible is `r`, since `""` would match `"abc.def.alice"`, which is wrong.